infra/aws/aws-organizations.md

```diff
@@ -63,11 +63,8 @@ description of the patterns we've adopted.
   billing data without having to give them direct access to the `org-root`
   account. *This cannot be changed once the account is created, so be sure
   to do it now.*
-* To access other accounts, users will use access keys in this account,
-  then assume a role in another account that allows access from this one.
-  See the [terraform-aws-iam-cross-acct-src](https://github.com/trussworks/terraform-aws-iam-cross-acct-src)
-  and [terraform-aws-iam-cross-acct-dest](https://github.com/trussworks/terraform-aws-iam-cross-acct-dest)
-  modules for the way we do this.
+* To access other accounts, we assign group policies which allow direct role assumption.
+  See the [terraform-aws-iam-cross-acct-dest](https://github.com/trussworks/terraform-aws-iam-cross-acct-dest) modules for the way we do this. See [`terraform-layout-example`](https://github.com/trussworks/terraform-layout-example) for an example of how this is done.
```

Comment:
nit: This sentence is a little jumbled -- you say "see this" twice in a row, and refer to a single module with a plural. I'd also mention the iam-user-group module -- saying "We use these modules to do this, see the layout example for how we use them" makes more sense to me.

Reply:
Or basically what you say in the other file. :)